Makefile

```diff
@@ -268,7 +273,10 @@ $Q +$(MAKE) --no-print-directory $(addprefix $(BUILD)/,$(1))
 endef
 
 .PHONY: build
-build: $(BUILD)/dummy ## ensure all packages build
+build: $(BUILD)/fmt ## ensure all packages build
+	go build ./...
+	$Q # caution: some errors are reported on stdout for some reason
+	go test -exec true ./... >/dev/null
```

Comment:
any reason for >/dev/null here?

Reply:
Just that it's quite noisy, printing every package at it builds it, and somewhat misleading as it looks like test runs but it didn't run any actual tests